zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/Interpreter.java

```diff
@@ -249,15 +249,22 @@ public Interpreter getInterpreterInTheSameSessionByClassName(String className) {
     private String name;
     //@SerializedName("interpreterClassName")
     private String className;
+    private boolean defaultInterpreter;
```

Comment:
Sure. I also think it makes users feel comfortable if we adopt aliases and change of UI to show binded interpreter on a paragraph. Actually, in case of ordering of groups, I hope we support a menu to select default group and interpreter per user easily. It also makes backend simple.

Finally, the first impression of this work is that we should make it easily to insert a new custom interpreter not having in a `zeppelin.interpreters`. And because I also don't want to break any current user experiences, it results making a different configuration values. For the future, I'll simplify these ones.